### PR TITLE
1. pairQA and ptQA implemented in pd tasks 2. ptmax increased in AliF…

### PR DESCRIPTION
**Pull request by sbhawani**
…emtoDreamTrackHist.cxx

### CODE REVIEW

File: PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackCuts.cxx [hunk 1]
```diff
@@ -12,122 +12,122 @@
...
+    fCrossedRows(70),
+    fRatioCrossedRows(0.83),
+    fCutPID(false),
+    fAllowITSonly(false),
+    fCutHighPtSig(false),
+    fParticleID(AliPID::kUnknown),
+    fNSigValue(3.),
+    fNSigValueITS(3.),
+    fPIDPTPCThreshold(0),
+    fRejectPions(false) {
```
Comment (bhohlweger) [inline PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackCuts.cxx]: Please don't reformat the whole class. Use e.g. this style file: https://github.com/bhohlweger/GentleFemto/blob/master/eclipse-cpp-google-style.xml

File: PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.cxx [hunk 2]
```diff
@@ -52,7 +52,7 @@ AliFemtoDreamTrackHist::AliFemtoDreamTrackHist(bool DCADist, bool CombSig, bool
       fMultRangeHigh(55) {
   TString sName[2] = { "before", "after" };
   float ptmin = 0;
-  float ptmax = 5;
+  float ptmax = 6.0;
   int ptBins = 100;
```
Comment (bhohlweger) [inline PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.cxx]: Also adjust the binning so the width of each bin stays the same

File: PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.cxx [hunk 3]
```diff
@@ -453,7 +453,7 @@ AliFemtoDreamTrackHist::AliFemtoDreamTrackHist(TString MinimalBooking)
 
   fpTDist[0] = 0;
   TString ptName = Form("pTDist_%s", "after");
-  fpTDist[1] = new TH1F(ptName.Data(), ptName.Data(), 200, 0, 5);
+  fpTDist[1] = new TH1F(ptName.Data(), ptName.Data(), 200, 0, 6.0);
```
Comment (bhohlweger) [inline PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.cxx]: Change the nBins accordingly so the binwidth stays the same

Comment (sbhawani) [reply PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.cxx]: Now I changed bin size accordingly

**Comment by bhohlweger** [comment]
+test

**Comment by bhohlweger** [comment]
+test

**Comment by a-mathis** [comment]
+1

**Pull request merged**